docs/desktop/gnome/flatpak.md

```diff
@@ -138,7 +138,7 @@ Memorizing the list of commands is not necessary, but knowing how to get to the
     ```text
     F: Failed to parse /var/lib/flatpak/appstream/flathub/x86_64/active/appstream.xml.gz file: Error on line 4065 char 29: <p> already set '
       Organic Maps is a free Android & iOS offline maps app for travelers,
-      tourists, hikers, drivers and cyclists.
+      tourists, hikers, drivers, and cyclists.
```

Comment:
Hahaha... That's in the output of that command, which is why I didn't correct it.